web/views.py

```diff
@@ -84,6 +86,81 @@ def form_valid(self, form):
     def get_success_url(self):
         return reverse('web:demographic-data-create')
 
+class ChildrenListView(generic.CreateView):
+    """
+    Allows user to view a list of current children and add children
+    """
+    template_name = 'web/children-list.html'
+    model = Child
+    form_class = forms.ChildForm
+
+    def dispatch(self, request, *args, **kwargs):
```

Comment:
I literally just did this wrong in my PR that hasn't been merged yet ... I'll go fix it. But the right way is to inherit from [this](https://ccbv.co.uk/projects/Django/1.11/django.contrib.auth.mixins/LoginRequiredMixin/) and CreateView.

It'll force them to login if they're not logged in.